cli/src/main/java/com/devonfw/tools/ide/commandlet/CreateCommandlet.java

```diff
@@ -0,0 +1,75 @@
+package com.devonfw.tools.ide.commandlet;
+
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.io.FileAccess;
+import com.devonfw.tools.ide.process.ProcessContext;
+import com.devonfw.tools.ide.process.ProcessResult;
+import com.devonfw.tools.ide.property.StringProperty;
+
+import java.nio.file.Path;
+
+/**
+ * {@link Commandlet} to create a new IDEasy instance
+ */
+public class CreateCommandlet extends Commandlet {
+
+  private final StringProperty newInstance;
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public CreateCommandlet(IdeContext context) {
+
+    super(context);
+    addKeyword(getName());
+    newInstance = add(new StringProperty("", false, "newInstance"));
+  }
+
+  @Override
+  public String getName() {
+
+    return "create";
+  }
+
```

Comment:
You need to override this so I can can call this commandlet outside of a project (IDE_HOME):
```suggestion

  @Override
  public boolean isIdeHomeRequired() {

    return false;
  }

```

Reply:
done